src/Template/Bake/Layout/default.ctp

```diff
@@ -13,4 +13,10 @@
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
 %>
+<?php
+
+/**
+  * @var \<%= $namespace %>\View\AppView $this
```

Comment:
In the context of a plugin this won't be `PluginName\View\AppView` it will still be `App\View\AppView`. It think you might need to always have the app's namespace available.